Index: appweb-4-4.4.1-0/configure
===================================================================
--- appweb-4-4.4.1-0.orig/configure	2013-10-03 15:27:56.000000000 +0000
+++ /dev/null	1970-01-01 00:00:00.000000000 +0000
@@ -1,23 +0,0 @@
-#!/bin/sh
-#
-#   configure -- Configure for building
-#
-#   Configure will invoke bit configure to prepare for building. If bit is not installed,
-#   configuration is not required. Variables can be passed to make to modify the build.
-#
-NAME=appweb
-OS=`uname | sed 's/CYGWIN.*/windows/;s/Darwin/macosx/' | tr '[A-Z]' '[a-z]'`
-PROFILE=debug
-EXT=mk
-
-if ! type bit >/dev/null 2>&1 ; then
-    echo '' >&2
-    echo "To configure ${NAME}, the \"bit\" tool is required. You can continue with a" >&2
-    echo 'default build by using make.' >&2
-    make help
-    exit 0
-fi
-if [ -f start.bit ] ; then
-    bit clean >/dev/null 2>&1
-fi
-bit configure "$@"
